wemake_python_styleguide/logic/prop_access.py

```diff
@@ -33,3 +35,20 @@ def parts(node: Union[ast.Attribute, ast.Subscript]) -> Iterable[ast.AST]:
         if chained_item is None:
             return
         iterator = chained_item
+
+
+def accesses(node: AnyAccess) -> Iterable[AnyAccess]:
+    """
+    Returns consecutive attribute and subscript accesses.
+
+    We get all parts from it except ``obj: ast.Name``.
+
+    .. code:: python
+
+      obj.attr.other[0].field
+
+    """
+    for part in parts(node):  # pragma: no cover, can't complete the loop
```

Comment:
No, sorry. `# pragma: no cover` is not acceptable here. It means that:
1. You either have insufficient test cases
2. Or you have wrong logic that never executes

This one needs to be fixed.